Lib/python/pyhead.swg

```diff
@@ -36,7 +36,7 @@
 SWIGINTERN char*
 SWIG_Python_str_AsChar(PyObject *str)
 {
-#if PY_VERSION_HEX >= 0x03030000
+#if PY_VERSION_HEX >= 0x03030000 && !defined(Py_LIMITED_API)
```

Comment:
Instead of this #ifdef, and reverting back to a Python 2 API, can you use both `PyUnicode_AsUTF8String()` and `PyBytes_AsString()`?  The ifdef would fall out, and the limited api would be supported.  That seems simple enough to commit/review/merge separately from a Py_LIMITED_API switch.

Reply:
Addressed by #2392